spec/formats/news_story_spec.rb

```diff
@@ -52,19 +57,39 @@ def then_i_can_publish_the_document
 
   def content_body
     {
-      "links" => {
+      "links" => a_hash_including(
```

Comment:
That's to not couple it to our app ordering. If I remember right I got caught on a failing test because of hash key ordering and that felt a bit too coupled.

Reply:
So I think it would be better to leave it as a plain old hash, because the 'including' matcher implies there could be other stuff in there, event though we know there isn't. I don't think we're relying on keys here, so it should be OK?